buildtest/cli/inspect.py

```diff
@@ -88,65 +94,118 @@ def fetch_test_names(report, names):
     return query_builders
 
 
-def inspect_list(report, terse=None, header=None, builder=None, color=None):
+def print_builders(report):
+    """This method prints all the builders.
+
+    Args:
+        report (str): Path to report file
+    """
+
+    builders = report.builder_names()
+    for name in builders:
+        console.print(name)
+
+
+def print_terse(table, no_header=None, consoleColor=None):
```

Comment:
oh we might as well be consistent with the variable names and not do camel case `consoleColor` this just doesn't look right. I think this was done by Jeff a while ago. Perhaps rename this variable to all lower case either `consolecolor` or `console_color`